pimcore/models/Asset/Text.php

```diff
@@ -25,4 +27,22 @@ class Text extends Model\Asset
      * @var string
      */
     public $type = "text";
+
+    public function getText($page = null)
+    {
+        if (preg_match("/\.?(txt|csv|xml)$/", $this->getFilename())) {
+            $cacheKey = "asset_text_text_" . $this->getId() . "_" . ($page ? $page : "all");
+
+            if (!$text = Cache::load($cacheKey)) {
```

Comment:
I don't think we need a cache here, reading from the file system should be sufficient. 
I'm also not sure if we need a check for the file extension, if the asset is classified as a text it shouldn't matter which file extension it has, or what is the idea behind that?